astropy/units/core.py

```diff
@@ -1328,14 +1327,38 @@ def cgs(self):
     @property
     def physical_type(self):
         """
-        Return the physical type on the unit.
+        Return the physical type(s) that correspond to a unit.
+
+        Returns
+        -------
+        ~astropy.units.physical.PhysicalType
+            A representation of the physical types of a unit.
 
         Examples
         --------
         >>> from astropy import units as u
-        >>> print(u.m.physical_type)
-        length
+        >>> u.m.physical_type
+        'length'
+
+        >>> u.Pa.physical_type
+        {'energy density', 'pressure', 'stress'}
```

Comment:
I strongly disagree. A Pascal is not a unit of energy density (J / m³). It just decomposes to the same SI base units.